### PR TITLE
chore(api): update to 0.18.1

### PR DESCRIPTION
**Pull request by thedoublejay**

### CODE REVIEW

File: src/subcommand/server.rs [hunk 1]
```diff
@@ -1660,85 +1637,60 @@ impl Server {
...
-        } else {
-          info.inscription.content_type().map(str::to_string)
-        }
-      } else {
-        info.inscription.content_type().map(str::to_string)
-      };
-
       Ok(if accept_json {
-        Json(api::Inscription {
-          address: info
```
Comment (thedoublejay) [inline src/subcommand/server.rs:1679]: @danadi7

File: src/index.rs [hunk 2]
```diff
@@ -1943,18 +1927,57 @@ impl Index {
...
+        next,
+        number: entry.inscription_number,
+        parents,
+        previous,
+        rune,
+        sat: entry.sat,
+        satpoint,
+        timestamp: timestamp(entry.timestamp.into()).timestamp(),
+        value: output.as_ref().map(|o| o.value),
+        // ---- Ordzaar ----
```
Comment (thedoublejay) [inline src/index.rs:1970]: here @danadi7

Comment (danadi7) [reply src/index.rs:1970]: lgtm

File: src/subcommand/wallet/batch_command.rs [hunk 3]
```diff
@@ -120,7 +120,14 @@ impl Batch {
 
     let current_height = u32::try_from(bitcoin_client.get_block_count()?).unwrap();
 
-    let reveal_height = current_height + 1 + u32::from(Runestone::COMMIT_INTERVAL);
+    let reveal_height = current_height + u32::from(Runestone::COMMIT_CONFIRMATIONS);
+
+    let first_rune_height = Rune::first_rune_height(wallet.chain().into());
```
Comment (danadi7) [inline src/subcommand/wallet/batch_command.rs:125]: is this our own additional check?

File: src/subcommand/server.rs (hunk 1 above)
Comment (danadi7) [reply src/subcommand/server.rs:1679]: Does this work without the custom Ordzaar changes?

**Pull request closed without merging**